ale_linters/powershell/powershell.vim

```diff
@@ -49,6 +49,11 @@ function! ale_linters#powershell#powershell#Handle(buffer, lines) abort
                 let l:matchcount = 1
             endif
 
+            " If the match is 0, it was a failed match
+            " probably due to an unexpected token which
+            " contained a newline. Reset matchcount and
+            " continue to next set of matches
+            if ! l:match[1] | let l:matchcount = 0 | continue | endif
```

Comment:
Write `if !empty(l:match[1])`, the `l:item` assignment below, and put `let l:matchcount = 0` in `else`. No need to put an if statement all in one line for script files like this, and no need to `continue` when there's not a lot of code below.